packages/core/src/character/character-transform-probe.ts

```diff
@@ -3,6 +3,7 @@ import { Object3D, PerspectiveCamera } from "three";
 import { CameraManager } from "../camera/camera-manager";
 
 export class CharacterTransformProbe {
+  private shouldUpdateHash: boolean = false;
```

Comment:
This seems to be just explicitly disabling the location hash behaviour. Is this intentional?

Reply:
Now collision detection on certain surfaces ends up with the auto-updating hash constantly changing due to the position precision used on the hash "encoding". I made it easy to turn it back on if you prefer to, but I'd suggest we have a UI element to share the encoded URL instead of updating it so often.